composer.json

```diff
@@ -6,11 +6,18 @@
   "license": "GPL-3.0-or-later",
   "prefer-stable": true,
   "minimum-stability": "dev",
-  "require": {},
+  "require": {
+    "php": ">=5.3|>=7.0"
+  },
   "require-dev": {
-    "phpunit/phpunit": "^5.6",
+    "phpunit/phpunit": "7.5.18",
     "wp-cli/wp-cli": "~1.5.1",
-    "woocommerce/woocommerce-sniffs": "0.0.8"
+    "woocommerce/woocommerce-sniffs": "0.0.9"
```

Comment:
0.0.9 version of WC sniffs require PHP 7.0, so when installing AS lib on PHP 5.6, I get this error:
```
> .\vendor\bin\phpcs classes

Parse error: syntax error, unexpected ':', expecting ';' or '{' in C:\Users\slaFFik\Sites\libs\woocommerce\action-scheduler\vendor\woocommerce\woocommerce-sniffs\src\WooCommerce-Core\Sniffs\Commenting\CommentTagsSniff.php on line 32
```